coalahtml/_coalahtml/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "devDependencies": {
+    "angular-mocks": "^1.5.0",
+    "chai": "^3.5.0",
+    "karma": "^0.13.21",
```

Comment:
Do you need to distribute packages required only to run tests? If it's possible to keeps them in the root package.json I think it would be nicer.